classes/class-briqpay-checkout.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Class for managing actions during the checkout process.
+ *
+ * @package Briqpay_For_WooCommerce/Classes
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class for managing actions during the checkout process.
+ */
+class Briqpay_Checkout
+{
+    public function __construct() {
+        add_action( 'woocommerce_after_calculate_totals', array( $this, 'update_briqpay_order' ), 9999 );
```

Comment:
Add another 9 here. I believe its smart coupons that run on the same priority and can cause a issue with us running updates before they apply their coupons